notebook/data_exploration/dataset_stats.ipynb

```diff
@@ -0,0 +1,262 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dataset Statistics\n",
+    "This notebook keeps track of various statistics about the compendium dataset like how many samples are labled, which diseases are present, etc.\n",
+    "\n",
+    "To skip straight to the stats, click [here](#stats)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from collections import Counter\n",
+    "import json\n",
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "# __file__ isn't defined for jupyter notebooks\n",
+    "curr_path = os.path.dirname(os.path.abspath(os.path.abspath('')))\n",
+    "whistl_path = os.path.join(curr_path, os.pardir, 'whistl')\n",
+    "sys.path.append(whistl_path)\n",
+    "import utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "map_file = os.path.join(curr_path, os.pardir, 'data', 'sample_classifications.pkl')\n",
+    "sample_to_label = utils.parse_map_file(map_file)\n",
+    "sample_ids = sample_to_label.keys()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Not all labeled samples show up in the compendium, which causes pandas to panic. To fix this we have to \n",
```

Comment:
Its been a while since I've reviewed this repo so excuse the obvious question -- so your "compendium" is your expression data that is being pulled from refine?  And your sample_ids that are mapped to labels for a certain disease is being pulled from where?  And there are fewer samples with labels then samples in your compendium which is why you are taking the intersection so that you have a set of samples with labels and expression?

Reply:
Not sure if this is something that will happen, but I ran into an issue with multiple distinct studies have the same sample_id.  But something that might throw off your labeled samples if that is the case

Reply:
The compendium is the refine.bio normalized human compendium of gene expression data. The sample_ids that are being mapped to labels come from the refine.bio metadata, the labels are done manually.
Since I pulled metadata from the refinebio GUI, the samples that are labeled are not a strict subset of the compendium data. This is because some samples don't meet the inclusion criteria/QC to become a part of the compendium.

Since the sample labeling is implemented as a dict, each sample you see here is unique. You are correct that the mapping from sample to study isn't one to one though.